### PR TITLE
Feature/stij 149

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/41-organisms/footer/_footer.scss [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
-
   .db-block-partnership {
     @include tablet {
       margin-top: 6rem;
     }
+  }
+}
+
+// Footer 2 column layout.
+footer[data-columns*="2"] {
```
Comment (Jeroen005) [inline components/41-organisms/footer/_footer.scss]: Should be fixed without the `*`

File: components/41-organisms/footer/footer.twig [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<footer class="section--wrapper" role="contentinfo">
+<footer class="section--wrapper" role="contentinfo" data-columns="{{ columns }}">
```
Comment (Jeroen005) [inline components/41-organisms/footer/footer.twig]: Should we provide some if statements so the col 2 and 3 vars aren't containing more than x cols?

Comment (meirege) [reply components/41-organisms/footer/footer.twig]: Isn't that the responsibility of the developer? We could do that, but it's not really useful imo...
If we add those to the style guide templates, it still doesn't stop them from implementing it in their own projects, seeing as we don't provide a plug and play type of style guide in terms of template files...
I would add it explicitly in the documentation instead.

File: components/41-organisms/footer/_footer.scss (hunk 1 above)
Comment (meirege) [reply components/41-organisms/footer/_footer.scss]: Done.

**Pull request merged**